src/main/scala/Test.scala

```diff
@@ -0,0 +1,11 @@
+/**
+  * Created by s on 3/12/17.
```

Comment:
Could you change your IntelliJ template to include your in scaladoc style? For example, I have

```
/**
  * @author Bor-Yuh Evan Chang
  */
```